server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -760,6 +760,13 @@ export function configureWebSocketServices(
 							// Keep track of disconnected clientIds so that we don't repeat the disconnect signal
 							// for the same clientId if retrying when connectDocument completes after disconnectDocument.
 							clientIdClientsDisconnected.add(clientId);
+						})
+						.catch((error) => {
+							Lumberjack.error(
+								"Failed to remove client from client manager",
+								undefined,
```

Comment:
It might be useful to pass in the properties like docid, tenantid, the clientId that we are trying to remove etc for debugging purpose.

Reply:
Done, thanks for the input!